.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Report a reproducible bug or regression.
+title: ''
+labels: 'type: bug 🐞'
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        🐞 If you do need to report a bug, please follow the template below to increase the chances of your report being looked at.
```

Comment:
follow the template below to increase -> please complete the template below as much as possible to increase